sources/entities/mediaplayer.cpp

```diff
@@ -146,6 +146,12 @@ void MediaPlayer::playMedia(const QString &itemKey, const QString &type) {
     map.insert("id", itemKey);
     command(MediaPlayerDef::C_PLAY_ITEM, map);
 }
+void MediaPlayer::addToQueue(const QString &itemKey, const QString &type) {
+    QVariantMap map;
+    map.insert("type", type);
+    map.insert("id", itemKey);
+    command(MediaPlayerDef::C_QUEUE, map);
```

Comment:
Same issue as with the spotify integration: MediaPlayerDef is part of the shared integrations.library project, which also requires a PR before new enums can be used in remote-software and the integrations ;-)
```
../sources/entities/mediaplayer.cpp:153:29: error: 'C_QUEUE' is not a member of 'MediaPlayerDef'
     command(MediaPlayerDef::C_QUEUE, map);
                             ^~~~~~~
```

Reply:
The required PR of integrations.library is merged and a new release v0.5.1 has been created.
Required change for this PR: modify dependencies.cfg to include the new release
```
integrations.library: v0.5.1
```